_component_design/popovers.md

```diff
@@ -18,36 +18,41 @@ helix-ui-javascript: true
 
 <section class="static-section" markdown="1">
 
-## Introduction
-
 <div class="hxRow"  markdown="1">
 
-{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
-
-### When to Use
+{% column left:"hxCol-12 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12" %}
 
-Popovers are triggered as a result of a user action and provide a user with additional context through language, actions through buttons, or both.
+## When to Use
 
-You can use popovers for short create or edit workflows and are particularly useful if context from the page is needed while making the change. Consider creating a new page if a large workflow is required.
+Popovers appear when a user interacts with an action link on a page. For example, when a user clicks an **Edit** link, a popover containing the parameters to edit opens.
```

Comment:
something to the effect of "Use a popover for small Create, Read, Edit, or Delete workflows where there is no need to change context."